types/picomatch/lib/picomatch.d.ts

```diff
@@ -0,0 +1,77 @@
+import constantsImport = require('./constants');
+import parseImport = require('./parse');
+import scanImport = require('./scan');
+import { PicomatchOptions as PicomatchOptionsImport } from './picomatch-options';
```

Comment:
We do we need to use as here?

Reply:
There is aliasing going on when we re-expose `PicomatchOptions` in the exposed namespace. `type PicomatchOptions = PicomatchOptions;` would be no bueno.